hypervisor/arch/x86/guest/vioapic.c

```diff
@@ -105,11 +105,11 @@ vioapic_send_intr(struct vioapic *vioapic, uint8_t pin)
 }
 
 static void
-vioapic_set_pinstate(struct vioapic *vioapic, uint8_t pin, bool newstate)
+vioapic_set_pinstate(struct vioapic *vioapic, uint32_t pin, bool newstate)
```

Comment:
using uint32_t may lead to many type casting in caller side, since we have decided to use uint8_t for pin....  Please check with Junjie on this.

Reply:
Yes. I was already checked with Junjie. He helped to scan the MISRA C, all warnings are already fixed in this patch set.

Reply:
ok